src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -429,3 +429,18 @@ module:io.netty.* -app
 module:newrelic -app -group
 path:**/*newrelic*/** -app -group
 path:**/*LogRocket*/** -app -group
+
+## Unreal Engine
+
+### Unreal Engine internal assertion handling
+family:native stack.function:FDebug::CheckVerifyFailedImpl* ^-app -app v+app ^-group -group v+group
+
+### Unreal Engine internal ensure handling
+family:native stack.function:UE::Assert::Private::ExecCheckImplInternal* ^-app -app v+app ^-group -group v+group
```

Comment:
[This](https://ivan-tustanivskyi.sentry.io/issues/6233966368/events/20510eada8c74b7e189913ad6152b29f/?project=4508533445754880) is an example of this rule:
<img width="930" alt="image" src="https://github.com/user-attachments/assets/66c38460-f593-4a30-925e-3eaab911f0fb" />

@tustanivsky there are 5 frames excluded from grouping that have source code (see the ones which can be expanded).

Should any of those be in-app? or part of grouping?

Reply:
I believe these frames represent internal Unreal SDK logic for processing ensure events and can be safely ignored.